src/transformers/training_args_tf.py

```diff
@@ -30,6 +30,12 @@ class TFTrainingArguments(TrainingArguments):
             "help": "Name of a Tensorflow loss. For the list see: https://www.tensorflow.org/api_docs/python/tf/keras/losses"
         },
     )
+    tpu_name: str = field(
+        default=None, metadata={"help": "Name of TPU"},
+    )
+    end_lr: float = field(
+        default=0, metadata={"help": "End learninig rate for optimizer"},
```

Comment:
```suggestion
        default=0, metadata={"help": "End learning rate for optimizer"},
```

Reply:
Not a huge fan of `args.end_lr` – in our PyTorch Trainer we can now pass a custom optimizer if we want to customize the optimizer further. But I guess we can keep it here for now